ActionSheetPicker-3.0.podspec

```diff
@@ -1,16 +1,17 @@
 Pod::Spec.new do | s |
     s.name = 'ActionSheetPicker-3.0'
-    s.version = '2.3.0'
+    s.version = '2.3.1'
     s.summary = 'Better version of ActionSheetPicker with support iOS7 and other improvements.'
     s.homepage = 'http://skywinder.github.io/ActionSheetPicker-3.0'
     s.license = 'BSD'
     s.authors = {
         'Petr Korolev' => 'https://github.com/skywinder',
         'Tim Cinel' => 'email@timcinel.com',
+        'Andrea Busi' => 'https.//github.com/bubidevs'
```

Comment:
To become the author, please refer to https://github.com/skywinder/ActionSheetPicker-3.0/issues/348 I would be happy to discuss it there. But don't put in in PR of podspec, please